domains/magdi.json

```diff
@@ -4,6 +4,17 @@
         "email": "ahmed.magdi@msn.com"
     },
     "record": {
-        "MX": ["mx01.mail.icloud.com"]
+        "MX": [
+            "mx01.mail.icloud.com",
+            "mx02.mail.icloud.com"
+        ],
+        "TXT": [
+            "apple-domain=MoEJzvyzzLKIBX7R",
+            "v=spf1 include:icloud.com ~all"
+        ],
+        "CNAME": [
+            "coeur85.github.io",
+            "sig1.dkim.magdi.is-a.dev.at.icloudmailadmin.com"
+        ]
```

Comment:
1. You can't have multiple CNAME records
2. You cannot have CNAME records with other records